src/data/community-meetups.json

```
@@ -1,4 +1,10 @@
 [
+    {
+    "title": "DC Onchain",
+    "emoji": ":us:",
+    "location": "DC",
```

@wackerow thoughts on location 🤔?

```suggestion
    "location": "Arlington, Virginia",
```